bundles/org.openhab.binding.intesis/README.md

```diff
@@ -0,0 +1,83 @@
+# Intesis Binding
+
+This binding connects to WiFi [IntesisHome](http://www.intesishome.com/) devices using their local REST Api.
+It does actually not support [IntesisBox](http://www.intesisbox.com/) devices but support is planned in upcoming version.
+
+
+
+## Supported Things
+
+This binding only supports one thing type:
+
+| Thing      | Thing Type | Description                                                            |
+|------------|------------|------------------------------------------------------------------------|
+| intesisHome | Thing      | Represents a single WiFi device                                         |
+
+## Discovery
+
+Intesis devices do not support auto discovery.
+
+## Thing Configuration
+
+The binding needs two configuration parameters, passwort and IP-Address.
+
+## Channels
+
+| Channel ID | Item Type          | Description                                                           | Possible Values |
+|------------|--------------------|-----------------------------------------------------------------------|-|
+| power      | Switch             | Turns power on/off for your climate system.                           | ON, OFF |
+| mode       | String             | The heating/cooling mode.                                             | AUTO, HEAT, COOL, DRY, FAN |
+| windspeed  | String             | Fan speed (if applicable)                                             | AUTO, 1-4 |
```

Comment:
Channels are handled as `Number` in your code. You might want to specify which number maps to which mode. Same for `swingUpDown`.

Reply:
Changed to Number, possible values accordingly. There will be no mapping explanation, as different AC Units may have different order of modes.